tests/test_cube/test_cube.py

```diff
@@ -231,11 +189,11 @@ def test_cube_cropping(tmpdir, loadsfile1):
     logger.info("Import SEGY format via SEGYIO")
 
     incube = loadsfile1
-
+    assert incube.dimensions == (408, 280, 70)
     # thinning to evey second column and row, but not vertically
     incube.do_cropping((2, 13), (10, 22), (30, 0))
-
-    incube.to_file(join(tmpdir, "cube_cropped.segy"))
+    assert incube.dimensions == (393, 248, 40)
```

Comment:
Maybe use `pytest-snapshot` here?

Reply:
Doesnt that defy the point somewhat? If I write it to file, the snapshot will be useful and human readable, but almost as large as before. I can compare it as bytes, but then the comparison if something changes becomes a nuisance.

Reply:
Agree :)